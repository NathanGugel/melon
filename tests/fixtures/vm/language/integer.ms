
func testInt |i| => {
    io.print(integer.fromString(i) ?? 0);
}

testInt("asdasd");
testInt("1");
testInt("1000");
testInt("15");
testInt("0");
testInt("+125");
testInt("-123124");