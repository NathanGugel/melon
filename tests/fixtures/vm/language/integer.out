0
1
1000
15
0
125
-123124
